packages/amplify-e2e-core/src/index.ts

```diff
@@ -99,10 +99,10 @@ export async function installAmplifyCLI(version: string = 'latest') {
   console.log('PATH SET:', process.env.AMPLIFY_PATH);
 }
 
-export async function createNewProjectDir(
+export const createNewProjectDir = async (
```

Comment:
Linting rules wanted const exports rather than function exports, so I did that a few places while I was troubleshooting my test setups.